test/src/main/scala/scalarules/test/twitter_scrooge/thrift/Thrift1.thrift

```diff
@@ -14,6 +14,6 @@ struct Struct1 {
 # which was moved between jdk8 and jdk11. So having this union is important for
 # testing jdk11, which requires the shims in javax.annotation:javax.annotation-api:1.3.2
 # to compile.
-union Union {
+union Union1 {
```

Comment:
Why is this needed?

Reply:
With the newer version of scrooge-generator, a name clash occurs.
I got the error running `bazel test test/src/main/scala/scalarules/test/twitter_scrooge/...`:
```
ERROR: /Users/etyurina/workspace/rules_scala_fork/test/src/main/scala/scalarules/test/twitter_scrooge/thrift/BUILD:3:15: scala //test/src/main/scala/scalarules/test/twitter_scrooge/thrift:thrift failed (Exit 1)
bazel-out/darwin-fastbuild/bin/test/src/main/scala/scalarules/test/twitter_scrooge/thrift/tmp4946029676322863219/scalarules/test/twitter_scrooge/thrift/Union.scala:85: error: value Struct1 is not a member of org.apache.thrift.protocol.TStruct
          _result = Union.Struct1({
                          ^
one error found
```
The generated file is [here](https://gist.github.com/cattibrie/8d8e42046255c26fe99328b0cee6ec91).

Reply:
Got you. Next time I think there's value in either adding a comment or better yet making the rename more explicit (maybe unionWhichIsntReservedWord or something).Thanks for the contribution!

Reply:
Thank you @ittaiz 
Will comment or give a better naming next time 🙂

Thank you for the help and review @ittaiz, @liucijus!

Reply:
Thank you for your time and effort! We definitely need an active thrift scrooge champion around here :)